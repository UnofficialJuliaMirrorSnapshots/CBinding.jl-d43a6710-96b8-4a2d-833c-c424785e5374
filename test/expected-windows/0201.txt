9: 000000000000000000
x: 030000000000000000
d: 00db6a760ae4fb7a88
