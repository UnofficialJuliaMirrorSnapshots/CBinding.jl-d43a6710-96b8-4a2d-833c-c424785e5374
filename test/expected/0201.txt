8: 0000000000000000
x: ea00000000000000
i: 00000000ea693b1a
